Automattic/jetpack
# Add deprecated to VideoPress block

Fixes Automattic/greenhouse#1028

#### Note

If the person in the Jetpack crew reviewing this PR has some knowledge on block "deprecated", it would be greatly appreciated so we can have proper confirmation, if not, I'll try to find someone on Slack to do a third review round before merging (so, please do not merge this PR if it gets accepted but you're not sure about the "deprecated" part :) )

#### Changes proposed in this Pull Request:

After statically adding cover=true to all URLs generated from the video block, Gutenberg block validation failed for blocks already created.
This PR adds a deprecated with the previous markup to safely migrate existing blocks.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* On a Jetpack site with VideoPress enabled
* Checkout Jetpack version 10.2 or earlier ( commit 2bfb4cc02f )
* jetpack build plugins/jetpack
* Create a new post, add a /video block, upload a video on VideoPress or select an existing one and save.
* Checkout Jetpack master branch and jetpack build plugins/jetpack
* ✅ Reload your post, validation should fail and the video block should appear as invalid
* Checkout this branch and jetpack build plugins/jetpack
* ✅ Reload your post, the block should appear as valid and be migrated/load the video
